packages/cascara/src/ui/ChatMessages/ChatAttachment.js

```diff
@@ -8,7 +8,7 @@ import {
   Ref,
 } from '@fluentui/react-northstar';
 import { FilesEmptyIcon } from '@fluentui/react-icons-northstar';
-import { Boundaries } from '../../system-components';
+import { Boundaries } from '../../atoms';
```

Comment:
And then one where we are importing a named export from the directory

Reply:
We should probably use the direct import, what do you think? @brianespinosa

Reply:
@lordpixel I agree then. Let's officially move to support only the direct import syntax.

I am going to approve this PR and have a task to remove all of the directory exports and have a singular export file with everything then. That will prevent anyone from doing anything but the direct export that we want from now on. Thank you for clarifying.